scalableminds/webknossos
# Make "Go to center of bounding box" available at one click

### URL of deployed dev instance (used for testing):
- https://centerbbsingleclick.webknossos.xyz/

### Steps to test:
- Open an annotation with some bounding boxes
- Click anywhere in a bounding box entry. This should center the bounding box within the view ports
- Editing the color, the name, the coordinates, clicking the context menu or any of its entries or switching the visibility shouldnt center the bbox
- the rest of the UI should be working as before

### TODOs:
- [x] editing the coordinates or the color shouldnt focus bbox

### Issues:
- fixes #8001

------
(Please delete unneeded items, merge only when none are left open)
